### ISSUE TITLE
[Bug]: Social Share doesn't appear on privacy features on

### ISSUE THREAD

**Issue by Tkkg1994** [stefanobartoletti/nuxt-social-share#314]
### Please take a moment to familiarize yourself with this project scope and standards

- [X] I have read the [Documentation](https://github.com/stefanobartoletti/nuxt-social-share?tab=readme-ov-file) and I understand how to correctly use this module.
- [X] I have read the [Contributing Guidelines](https://github.com/stefanobartoletti/nuxt-social-share/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow this project's [Code of Conduct](https://github.com/stefanobartoletti/nuxt-social-share?tab=coc-ov-file)

### Environment

```shell
------------------------------
- Operating System: Windows_NT
- Node Version:     v20.15.1
- Nuxt Version:     3.14.1592
- CLI Version:      3.16.0
- Nitro Version:    2.10.4
- Package Manager:  npm@10.9.0
- Builder:          -
- User Config:      default
- Runtime Modules:  @pinia/nuxt@0.8.0, @nuxtjs/tailwindcss@6.12.2, @nuxtjs/i18n@9.1.0, @nuxtjs/seo@2.0.2, nuxt-headlessui@1.2.0, nuxt-vuefire@1.0.4, @nuxtjs/color-mode@3.5.2, @vee-validate/nuxt@4.14.7, @nuxt/eslint@0.7.2, pinia-plugin-persistedstate/nuxt, @unlok-co/nuxt-stripe@4.0.1, @nuxtjs/turnstile@0.9.11, @stefanobartoletti/nuxt-social-share@1.2.0
- Build Modules:    -
------------------------------
```

### Minimal reproduction link

[Link](https://stackblitz.com/edit/github-pp1ixx47?file=README.md)

### Describe the bug/issue

**Hello dear @stefanobartoletti** 

This is an awesome module. Thank you very much for prodiving it. Really enjoying it a lot 👍 

Now let us get into my small but critical problem 😆 

Many browsers have protections, adblockers or similar included. That can be an extension or even built into the browser

### Example: Braves privacy feature.

if turned on, my sharing looks like this:
![image](https://github.com/user-attachments/assets/20a78f74-d2c6-4c19-a02f-17c2f03e5c39)

Without the protection enabled:
![image](https://github.com/user-attachments/assets/166cbcf0-01bb-48a5-95aa-759fe995b32e)

Same thing goes for adblockers reported by users on any browser.

I can still checkout the html in the devtools and see that's there, just not visible.

### Potential solutions
- Renaming the classes to not include ad sounding titles
- Not get detected as ad (no idea how)
- Whitelist options somewhere?
- If the share buttons can't be displayed give us a false prop which we can use to not display an empty share title

What do you think about this? I'm more then happy to help but yeah, don't know if we find a nice solution for this.

Keep up your awesome work! :

### Browsers

Chrome, Firefox, Brave, Safari, Microsoft Edge

### Logs

```shell
Just to include my config:
  socialShare: {
    baseUrl: baseUrl,
    styled: true,
    label: false
  },
```

### Additional context

`
<SocialShare
                  v-for="network in ['facebook', 'x', 'linkedin', 'whatsapp', 'telegram']"
                  :key="network"
                  :network="network"
                  class="text-white p-3 rounded-md shadow-sm hover:opacity-80 transition"
                />`

**Comment by stefanobartoletti** [comment]
Thanks for reporting, but with a quick test on both Brave (my main browser, so I should have noticed already), and Firefox with UBlocker, I cannot replicate this issue.

The component is rendered as simple HTML and the share link is just a simple anchor without any external Javsacript.

**Comment by stefanobartoletti** [comment]
You should provide a minimal reproducible demo.

And maybe check if with your settings you can see and use the social share buttons on the main documentation website https://nuxt-social-share.stefanobartoletti.it/

![Screenshot_20241202_125300](https://github.com/user-attachments/assets/70aa0dc4-5362-49e6-a798-8deacc27804c)

**Comment by Tkkg1994** [comment]
Hello dear @stefanobartoletti 

On your website, it looks like this on my browser:

![image](https://github.com/user-attachments/assets/d14116f6-178e-45d9-8858-72dcc8b17f62)

I can add a reproducible demo, but as it seems it's an issue with the component combined with the browser settings and this is hard to reproduce.

As it will work just fine for you as soon as you allow tracking yourself in browser

**Comment by stefanobartoletti** [comment]
Thanks for the extra info. 

I still don't know how to solve or even investigate this, the component renders basically only a link

```html
<a class="social-share-button social-share-button--facebook social-share-button--styled text-white border-none" href="https://www.facebook.com/sharer/sharer.php?u=https://nuxt-social-share.stefanobartoletti.it/" aria-label="Share with Facebook" target="_blank" style="--color-brand: #0866FF;">
    <svg class="social-share-button__icon" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img" width="1em" height="1em" viewBox="0 0 24 24"><path fill="currentColor" d="M14 13.5h2.5l1-4H14v-2c0-1.03 0-2 2-2h1.5V2.14c-.326-.043-1.557-.14-2.857-.14C11.928 2 10 3.657 10 6.7v2.8H7v4h3V22h4z"></path></svg>
    <span class="social-share-button__label">Share</span>
</a>
```

This is all, no extra scripting or anything else.

And unfortunately I can't reproduce it by using both Brave and Firefox with Ublock addon, on both desktop and mobile devices.

**Comment by stefanobartoletti** [comment]
Yes, I understand what you are saying but the issue is not reproducible. I have the same exact settings and everything works correctly here.

**Comment by Tkkg1994** [comment]
I am here to help you with anything I can, to provide you with the logs or information you need to resolve this. Because I would really love to get this to work.

I can send you videos, we can even do a teamviewer session. I am up for everything

**Comment by stefanobartoletti** [comment]
As explained in the issue template, I need a *minimal reproducible* setup, which is a required and not optional step. 

I'm only available to fix issues inherent to the module itself, not implementations on third-party projects, and only here through GitHub.

**Comment by Tkkg1994** [comment]
@stefanobartoletti 

Sorry it took so long, the holidays were quite intense over here.. so I finally created a stackblitz reproduction project:
https://stackblitz.com/edit/github-pp1ixx47?file=app.vue

Same here I see nothing when I enable the tracking projection:
![image](https://github.com/user-attachments/assets/5359bda2-a83f-4fee-b9f1-26cdff902437)


When disabled:
![image](https://github.com/user-attachments/assets/9c6e2ee6-8d85-4a3c-acf6-f32b827065e5)

Working fine. I hope this helps now!

Wish you a good rest of the week

**Comment by stefanobartoletti** [comment]
I'm using the same settings as yours, and the issue is not present. 

I tested it by using concurrently Brave privacy settings, "Ublock Origin" and "DuckDuckGo Privacy Essentials", and the buttons are always displayed.

I still cannot reproduce it and to be honest, it does not look like the issue originated from this module.

Try using different combination of browsers/OS/network/firewall

![Screenshot_20250110_172901](https://github.com/user-attachments/assets/de6f67bd-a339-41c3-b06f-fa4492112b10)

**Comment by Tkkg1994** [comment]
@stefanobartoletti

I really don't know what else we can try since we use the same browser and settings. But I pinned it down not to the adblocker but to the tracking settings being blocked

**Comment by stefanobartoletti** [comment]
Closing this as not reproducible
